src/components/button/button.stories.js

```diff
@@ -68,22 +56,22 @@ const generateButtons = buttonType => () => {
     <>
       {OptionsHelper.buttonIconPositions.map(iconPosition => (
         ['', ...OptionsHelper.icons].map((iconType) => {
-          const props = { iconPosition, buttonType, iconType };
+          const internalProps = { iconPosition, buttonType, iconType };
```

Comment:
We can change props to be `let props = getKnobs()`

```suggestion
          props = {...props, iconPosition, buttonType, iconType };
```

Reply:
I separated props on purpose to avoid knob crossover/conflicts between examples.